client/js/modalContent.js

```diff
@@ -0,0 +1,78 @@
+const getTitleAndDateFromModal = async() => {
+    /** 
+     *Funtion to read the title and data from the modal DOM and inject new rating div if one doesn't exist
+     * @param {}
+     * @return {}
+    */
+
+    // Reads the title name of movie / series from the previewModal--player-titleTreatment-logo class
+    let titleElement = document.querySelectorAll('.previewModal--player-titleTreatment-logo')
+    let title = (titleElement === undefined  || titleElement[0] === undefined ) ? null : titleElement[0].getAttribute("alt")
+
+    if(!title || title == null) return
```

Comment:
Check if this is needed.

Reply:
This is needed because if attribute `alt` does not exist then `titleElement[0].getAttribute("alt")` would return `null`